src/lib/permissions/spaces/client/spacePermissionsHttpClient.ts

```diff
@@ -32,4 +37,10 @@ export class SpacePermissionsHttpClient extends AbstractPermissionsApiClient imp
       headers: this.jsonHeaders
     });
   }
+
+  toggleRequireProposalTemplate(request: SpaceRequireProposalTemplateToggle): Promise<Space> {
```

Comment:
why is config being done in the permissiosn api? it should be fine to just put this in the webapp cc @motechFR